src/ResourceManager/KeyVault/Commands.KeyVault/Commands/ManagedStorageAccounts/SetAzureKeyVaultManagedStorageSasDefinition.cs

```diff
@@ -0,0 +1,361 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System;
+using Microsoft.Azure.Commands.KeyVault.Models;
+using System.Collections;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Management.Automation;
+using System.Text;
+using System.Xml;
+
+namespace Microsoft.Azure.Commands.KeyVault
+{
+    [Cmdlet( VerbsCommon.Set, CmdletNoun.AzureKeyVaultManagedStorageSasDefinition,
+        SupportsShouldProcess = true,
+        HelpUri = Constants.KeyVaultHelpUri,
+        DefaultParameterSetName = ParameterSetRawSas )]
+    [OutputType( typeof( ManagedStorageSasDefinition ) )]
+    public partial class SetAzureKeyVaultManagedStorageSasDefinition : KeyVaultCmdletBase
```

Comment:
This is a PUT.

---
In reply to: [119731517](https://github.com/Azure/azure-powershell/pull/4061#discussion_r119731517) [](ancestors = 119731517)